pkg/runtime/framework/plugins/torch/torch.go

```diff
@@ -60,13 +60,14 @@ func (t *Torch) Validate(runtimeInfo *runtime.Info, _, newObj *trainer.TrainJob)
 
 	specPath := field.NewPath("spec")
 
-	if newObj.Spec.Trainer != nil {
+	if newObj.Spec.Trainer != nil && newObj.Spec.Trainer.NumProcPerNode != nil {
```

Comment:
Why do we need `newObj.Spec.Trainer.NumProcPerNode != nil`? We already checked this conditional in Line 57